include/xnnpack.h

```diff
@@ -423,11 +432,12 @@ enum xnn_status xnn_define_channelwise_quantized_tensor_value_v2(
 ///                     number of input channel element per output channel.
 ///                     For Fully connected operators with 2d filters of size [output_channels, input_channels],
 ///                     expecting number of scale values to be = output_channels * (input_channels / block_size).
+/// @param flags - binary features of the Value. Supported values are XNN_VALUE_FLAG_BF16_SCALES.
```

Comment:
Please add more documentation to make it crystal clear for someone who has no idea about internal implementations.